.gitignore

```diff
@@ -68,3 +68,6 @@ Release/
 ap_gen_tool_release/
 APDOOM.WAD.bak
 APHERETIC.WAD.bak
+
+# Clangd cache
+.cache/
```

Comment:
Should generate cmake out of source, how is this required then? not sure.

Reply:
iirc that folder was created automatically in vscode due to the clangd extension (OSS alternative to intellisense). Depending on your development environment it may or may not be needed.